src/flux/task-plan.coffee

```diff
@@ -8,20 +12,35 @@ TaskPlanConfig =
     @_local[planId].settings ?= {}
     @_local[planId].settings.page_ids ?= []
     @_local[planId].settings.exercise_ids ?= []
+    @_local[planId].settings.exercises_count_dynamic ?= TUTOR_SELECTIONS.default
+
     #TODO take out once TaskPlan api is in place
     _.extend({}, @_local[planId], @_changed[planId])
 
   FAILED: ->
 
+  updateTutorSelection: (id, direction) ->
+    plan = @_getPlan(id)
+    {page_ids, exercise_ids, exercises_count_dynamic, description} = plan.settings
+    exercises_count_dynamic += direction
+
+    if (exercises_count_dynamic > TUTOR_SELECTIONS.max)
```

Comment:
You can use `exercises_count_dynamic = Math.min(exercises_count_dynamic, TUTOR_SELECTIONS.max)`